src/components/organisms/DataTable/BackendDataTable.vue

```diff
@@ -12,6 +12,7 @@
 					@fire-action="fireAction"
 				/>
 			</div>
+			<div class="wrapper">
```

Comment:
please rename the class. Otherwise there will be style conflicts :(
```suggestion
			<div class="table-content-wrapper">
```